authentication/oidc/oidc.go

```diff
@@ -108,7 +111,7 @@ func (a *Authenticator) Authenticate(request *http.Request) (*authentication.Use
 	if authorizationHeader == "" {
 		return nil, errors.New("Missing authorization header")
 	}
-	token := strings.TrimPrefix(strings.ToLower(authorizationHeader), "bearer ")
+	token := strings.TrimPrefix(authorizationHeader, "Bearer ")
```

Comment:
The idea here was that when you do `strings.ToLower(authorizationHeader)`, this would lowercase the entire authorization header including the token string itself. The token string contains uppercase letters and lowercasing them would make the verification of the token fail, due to the fact that it's no longer in it's original state.

That is why the lowercasing was removed.